scripts/example-gallery-builder/update-example/templates-react-16/package.json

```diff
@@ -22,7 +22,7 @@
     "lodash": "^4.17.21",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
-    "sass": "^1.67.0"
+    "sass": "^1.67.0"PKG_DEPS
```

Comment:
I'm guessing this was unintentionally added from the update-gallery build script?